doc/content/integrations/cloud-integrations/blockbax/_index.md

```diff
@@ -19,133 +19,98 @@ Check the [Blockbax documentation page](https://blockbax.com/docs/) for more inf
+        if (value != null) {
+            context.addMeasurement(subjectExternalId + "$" + metricExternalId, value, timestamp);
+        }
+    }
+}
+```
 
-Make sure your device is added to {{% tts %}}. See [Adding Devices]({{< ref "/devices/adding-devices" >}}) for more info. Many [Device Repository]({{< ref "/integrations/payload-formatters/device-repo" >}}) payload formatters output payloads such that it can be directly ingested by the Blockbax Platform. If this is not the case, then please follow the instruction below to create a custom payload formatter.
+Optionally you can check 'Automatically create subjects'. This will create a subject automatically if a subject does not exist.
 
-### Create payload formatter
+## Create a webhook in {{% tts %}}
 
-On the left menu of your {{% tts %}} application, select **Payload formatters &#8594; Uplink**.
+Use the Blockbax [Webhook template]({{< ref "/integrations/webhooks/webhook-templates" >}}) to create a Webhook integration on {{% tts %}}. Select **Integrations &#8594; Webhooks** on the left hand menu. Click **Add webhook** and select the **Blockbax** tile.
 
-Select **Javascript** payload formatter type.
+{{< figure src="webhook.png" alt="Setting up the webhook" >}}
 
-Paste the following contents in the **Formatter parameter** window:
+Enter an arbitrary **Webhook ID**, enter your **[Access token](https://blockbax.com/docs/project-settings/#access-tokens)** and **Endpoint**. The endpoint you can copy from the inbound connector settings.
 
-```javascript
-function decodeUplink(input) {
-    decoded = decoder(input.bytes)
-    return {
-        data: {
-            // For numeric metrics
-            <Blockbax-external-metric-ID>: decoded.exampleNumber,
-
-            // For text metrics
-            <Blockbax-external-metric-ID>: decoded.exampleText,
-
-            // For location metrics
-            <Blockbax-external-metric-ID>: {
-                lat: decoded.exampleLatitude,
-                lon: decoded.exampleLongitude
-            }
-        }
-    };
-}
+## Further set up your Blockbax project
 
-function decoder(bytes) {
-    // Add the logic that decodes the bytes from your device.
-    return decodedBytes
-}
-```
+Once you've added the webhook, payload are sent to Blockbax. Make sure your {{% tts %}} device is connected and has sent at least one payload. Now we are going to create a subject type which matches your device type(s) and further align your inbound connector if necessary. As an example we use the Oyster from Digital Matter, but the process is similar for other devices. 
 
-The functions presented above need to be modified against your setup. See example below.
+When you expand the executions in your inbound connector you should see something similar to the execution logs below.
 
-<details><summary>Uplink payload formatter example</summary>
+{{< figure src="execution-logs" alt="Example execution logs from an Oyster device from Digital Matter" >}}
```

Comment:
```suggestion
{{< figure src="execution-logs.png" alt="Example execution logs from an Oyster device from Digital Matter" >}}
```

Reply:
Good one, thanks for the suggestion. Committed!